conjur/cli.py

```diff
@@ -181,21 +204,18 @@ def run_client_action(resource, args):
         Helper for creating the Client instance and invoking the appropriate
         api class method with the specified parameters.
         """
+        if resource == 'init':
+            Client.setup_logging(Client, args.debug)
```

Comment:
Is `Client` first parameter is mandatory?
I think you can remove it.
```suggestion
            Client.setup_logging(args.debug)
```